<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreSet</class>
 <widget class="QWidget" name="PreSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>预设置</string>
  </property>
  <widget class="QGroupBox" name="tableGroup">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>表格设置</string>
   </property>
   <widget class="QGroupBox" name="LoadGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>162</width>
      <height>94</height>
     </rect>
    </property>
    <property name="title">
     <string>加载</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="rowCount">
       <property name="text">
        <string>行：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="rowLine"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="columnCount">
       <property name="text">
        <string>列：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="columnLine"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="opWidget" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>184</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okBtn">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
